crates/core/src/network/connection_manager.rs

```diff
@@ -567,7 +567,7 @@ impl ConnectionTracking for ConnectionManager {
             })
             .or_insert(new_status);
         if !is_already_gone {
-            info!(peer = %node_id, "Peer shutdown notification received");
+            debug!(peer = %node_id, "Peer shutdown notification received");
```

Comment:
Did you find this one noisy? It should be logged once per node.

Reply:
Then maybe we should define, what "info" should contain and who the typical user for info log is? There is no universal rule, and we need to have some anchoring point for us.

Reply:
Then let's keep it as is and get back to define what "info" should contain since this will be helpful for the long run and the others as well.